examples/oapp/layerzero.config.ts

```diff
@@ -21,6 +21,9 @@ const config: OAppOmniGraphHardhat = {
     contracts: [
         {
             contract: fujiContract,
+            config: {
+                callerBpsCap: BigInt(1000),
+            },
```

Comment:
Should callerBpsCap be specified at the node level or the edge level in the connections further down in layerzero.config.ts? I placed it at the node level because the [pattern for configureOftBps](https://github.com/stargate-protocol/stargate-v2/blob/825d447ead52725cc128751db8d58cf823899865/packages/stg-evm-v2/devtools/src/oft-wrapper/config.ts#L16) configures it at the node level

Reply:
Tested this out by deploying and wiring real contracts, then fetching the value of `callerBpsCap` to ensure it matches the value specified in the `layerzero.config.ts` -- works as expected